looplib/looptools.py

```diff
@@ -1,5 +1,9 @@
 from __future__ import division, print_function
+from scipy.spatial import KDTree
+
 import numpy as np
+import networkx as nx
```

Comment:
if reqs are getting heavier, should they be factored into a requirements.txt read by setup.py?

Reply:
sounds reasonable!